src/langapi/language_file.cpp

```diff
@@ -169,14 +173,19 @@ bool language_filest::generate_support_functions(
 
   for(auto &file : file_map)
   {
-    if(languages.insert(file.second.language->id()).second)
-      if(file.second.language->generate_support_functions(symbol_table))
+    auto &language = *file.second.language;
```

Comment:
This change seems unrelated to "Pass object factory parameters through to object factories"?